doc/modules/tree.rst

```diff
@@ -508,6 +516,15 @@ Mean Squared Error:
 
     H(X_m) = \frac{1}{N_m} \sum_{i \in N_m} (y_i - \bar{y}_m)^2
 
+Half Poisson deviance:
+
+.. math::
+
+    \bar{y}_m = \frac{1}{N_m} \sum_{i \in N_m} y_i
```

Comment:
I tried that one, but again, it doesn't make the formulae better or less wrong. Separate issue/PR?

Reply:
it's more about consistency than it is about correctness I think. Do we really need to use R_m? It seems we could simply rely on X_m and use N_m purely as a number and not as a set of indices.

Reply:
But yes I'm happy to have this in another PR. It's just that it might delay this one a bit more. So if not too many changes are involved, it seems that doing it here would be more efficient, but you probably have a better overview so it's your call ;)